src/test/scala/com/microsoft/hyperspace/index/DeltaLakeIntegrationTest.scala

```diff
@@ -48,6 +48,7 @@ class DeltaLakeIntegrationTest extends QueryTest with HyperspaceSuite {
   }
 
   before {
+    hyperspace = new Hyperspace(spark)
```

Comment:
why is this change needed (guide comment would help)?

Reply:
Sorry I forgot to remove debug code. 😅